spec/integration/rdf_migration_spec.rb

```diff
@@ -2,29 +2,53 @@
 
 describe "Mirating RDF terms" do
 
-  let(:mover) do
```

Comment:
You should leave `mover` up here and just define `file` as the first parameter. In each context do `let(:file) { FedoraMigrate.source.connection... }`